src/controllers/todo.php

```diff
@@ -0,0 +1,58 @@
+<?php
+include_once '../database/db.php';
+include_once '../utils/validators.php';
+include_once '../utils/session.php';
+include_once '../services/user.php';
+include_once '../services/todo.php';
+
+$status = $_POST['status'];
+$title = $_POST['title'];
+$description = $_POST['description'];
+$planned_start_date = $_POST['planned_start_date'];
+$created_at = $_POST['created_at'];
+$planned_end_date = $_POST['planned_end_date'];
+$actual_start_date = $_POST['actual_start_date'];
+$actual_end_date = $_POST['actual_end_date'];
```

Comment:
These this are not general for the controller. Because we don't only use the $_POST. So it is better to get these informations in every section rather than making them global